doc/Language/5to6-perlop.rakudoc

```diff
@@ -44,13 +44,16 @@ my $list   = (1,);     # a List in a scalar container
 
 =head2 The arrow operator
 
-As you typically will not be using references in Raku, the arrow is
-probably less useful as a dereferencing operator. If you do need to
-dereference something, however, the arrow is the dot. It is also the dot
-for method calls. So, Perl's C«$arrayref->[7]» becomes
-C<$arrayref.[7]> in Raku and, similarly, C«$user->name» becomes
-C<$user.name>. The C«=>» arrow is used for constructing L<C<Pair>|/type/Pair>s, see
-L<Pair term documentation|/language/terms#Pair>.
+For the Perl C«=>» arrow, see the 
+L<Comma operator|/language/5to6-perlop#Comma_operator>.
+
+The dereferencing operator, C«->», is replaced by the dot, which is
+also used for method calls.  So Perl's C«$arrayref->[7]» becomes
+Raku's C<$arrayref.[7]> and C«$user->name» becomes C<$user.name>.
+Note that dereferencing is rare in Raku.
```

Comment:
Perhaps add: and that `$arrayref[7]` is also valid syntax for positional access (losing the dot).

Reply:
I do not understand this well.  A few times, IIRC, I have needed a dot to get an array index to work.  I am fairly sure  that it was in `map` 's  Callable parameter.  So there is something like indirection that comes up rarely.  JJ seems to be making that distinction.

If that distinction is real, your addition would be better placed with 5to6's guidance about Arrays.

I'll try checking the dot-reference-op's sources, with modest hope of success.